asserts/account_key.go

```diff
@@ -62,6 +62,24 @@ func checkSinceUntilWhat(m map[string]any, what string) (*sinceUntil, error) {
 	}, nil
 }
 
+func checkValidSinceUntilWhat(m map[string]any, what string) (*sinceUntil, error) {
+	since, err := checkRFC3339DateWhat(m, "valid-since", what)
+	if err != nil {
+		return nil, err
+	}
+
+	until, err := checkRFC3339DateWhat(m, "valid-until", what)
```

Comment:
This is more of a spec question (and probably it was already raised, I just don't remember) but other assertions that have similar header fields use just "since" and "until". Do you remember why we're diverging from that?